chapter_natural-language-processing-pretraining/word2vec-pretraining.md

```diff
@@ -26,47 +37,77 @@ We will implement the skip-gram model by using embedding layers and minibatch mu
 As described in :numref:`sec_seq2seq`,
 The layer in which the obtained word is embedded is called the embedding layer, which can be obtained by creating an `nn.Embedding` instance in high-level APIs. The weight of the embedding layer is a matrix whose number of rows is the dictionary size (`input_dim`) and whose number of columns is the dimension of each word vector (`output_dim`). We set the dictionary size to $20$ and the word vector dimension to $4$.
 
-```{.python .input  n=15}
+```{.python .input}
 embed = nn.Embedding(input_dim=20, output_dim=4)
 embed.initialize()
 embed.weight
 ```
 
+```{.python .input}
+#@tab pytorch
+embed = nn.Embedding(num_embeddings=20, embedding_dim=4)
+print(f"Parameter embedding_weight ({embed.weight.shape}, "
```

Comment:
Let's consistently use single quote in f-string

Reply:
i'll fix this

Reply:
Sure! I missed that, thanks for the fix :)